features/step_definitions/page_steps.rb

```diff
@@ -242,27 +243,27 @@
 
 Then "I there should be one version in the history panel" do
   within '.mercury-panel .versions' do
-    page.should have_selector 'li', count: 1
+    expect(page).to have_selector 'li', count: 1
   end
 end
 
 Then "I should see the old version" do
   within_frame 'mercury_iframe' do
-    within '[data-mercury]' do
-      page.should have_content('This is a published page')
+    within "section[data-mercury='full']" do
+      expect(page).to have_content('This is a published page')
     end
   end
 end
 
 Then "there should be an indication of unpublished changes" do
   step %{I wait for ajax requests to complete}
-  within '.mercury-toolbar' do
-    page.should have_selector '.mercury-publish-button.unpublished'
+  within '.mercury-primary-toolbar' do
+    expect(page).to have_selector '.mercury-publish-button.unpublished'
   end
 end
 
 Then "there should not be an indication of unpublished changes" do
-  within '.mercury-toolbar' do
-    page.should_not have_selector '.mercury-publish-button.unpublished'
+  within '.mercury-primary-toolbar' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.